src/index.html

```diff
@@ -7,21 +7,377 @@
+          <a
+            class="logo"
+            href="#"
+          >
+            <img
+              src="images/logo.png"
+              alt="logo"
+            />
+          </a>
+
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link is_active"
+                >
+                  Apple
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Samsung
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Smartphones
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                  data-qa="nav-hover"
+                >
+                  Laptops & Computers
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Gadgets
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Tablets
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Photo
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Video
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+
+    <main class="main">
+      <section class="catalog">
+        <div class="catalog__content">
+          <div
+            class="card"
+            data-qa="card"
+          >
+            <img
+              src="./images/imac.jpeg"
+              alt="card image"
+              class="card__image"
+            />
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
+            <p class="card__code">Product code: 195434</p>
+            <div class="card__rating rating">
+              <div class="rating__stars stars">
+                <div class="stars__star stars__star--active"></div>
+                <div class="stars__star stars__star--active"></div>
+                <div class="stars__star stars__star--active"></div>
+                <div class="stars__star stars__star--active"></div>
```

Comment:
```suggestion
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
```
Consider using classes from the previous task